packages/base/SkillCard/43f62768-cbdb-401a-8130-30e12a29cf0d.json

```diff
@@ -0,0 +1,17 @@
+{
+  "data": {
+    "type": "card",
+    "attributes": {
+      "instructions": "If the user wants the data they see edited, AND the patchCard function is available, you MUST use the \"patchCard\" function to make the change. If the user wants the data they see edited, AND the patchCard function is NOT available, you MUST ask the user to open the card and share it with you, If you do not call patchCard, the user will not see the change. You can ONLY modify cards shared with you, if there is no patchCard function or tool then the user hasn\\'t given you access NEVER tell the user to use patchCard, you should always do it for them.",
```

Comment:
The way the instructions were bulleted in their previous location made them easier to read. Should we consider doing that here?

Reply:
Is this [e24d427](https://github.com/cardstack/boxel/pull/1444/commits/e24d4270dee44974fee99a96eae660bf39fcaa3e) what you mean? I need to update the instructions field as markdown so we could style the text with bullet.

<img width="575" alt="Screenshot 2024-07-23 at 08 14 31" src="https://github.com/user-attachments/assets/467a1eba-2765-4312-8d00-77ca0bd4159d">

Reply:
As long as @IanCal feels OK about how markdown will work as prompt input, it seems good to me 👍

Reply:
Yeah I don't see it causing a problem, its very familiar with markdown, that's mostly how we want it responding anyway and the majority of markdown written stuff is the same as you'd do it for raw text.

Only risks I can see are people using images (which won't be used) or getting poor performance with tables, but I think that's a small risk for a much better editing experience.

Reply:
I think it did cause a problem:

![Image 2024-07-29 12-20-44](https://github.com/user-attachments/assets/e003cbec-f44b-4cd9-9bb4-8160be38a768)

That logging is from a branch I deployed to track down the problem, it adds a try/catch around [this line](https://github.com/cardstack/boxel/blob/5a36839751b0e971223f1b5f9e3e74275ae9ea60/packages/ai-bot/helpers.ts#L55).

I’m not seeing the error now though, puzzlingly